deploy/deploy.mk

```diff
@@ -28,6 +28,10 @@ deploy-db:
 	podman exec -it flightctl-db psql -c 'ALTER ROLE admin WITH SUPERUSER'
 	podman exec -it flightctl-db createdb admin || true
 
+deploy-rabbitmq:
```

Comment:
Let's call it "deploy-mq".  Rabbitmq is an implementation detail.  Similar to "deploy-db" instead of "deploy-postgres".